geniza/corpus/templates/corpus/document_list.html

```diff
@@ -43,7 +43,7 @@ <h1 class="sr-only">{{ page_title }}</h1>
         </fieldset>
 
         <fieldset id="filters" aria-expanded="false">
-            <a href="#" role="button" id="close-filters-modal">
+            <a href="#" role="button" id="close-filters-modal" data-search-target="closeModal">
```

Comment:
Can we name `closeModalFilters` (`closeFilterModal`? or even just `closeFilters`? )? Would like a little more context so this is more immediately obvious; and it's possible we'll have other modals.